pkg/limayaml/limayaml.go

```diff
@@ -208,6 +208,7 @@ type PortForward struct {
 type CopyToHost struct {
 	GuestFile string `yaml:"guest,omitempty" json:"guest,omitempty"`
 	HostFile  string `yaml:"host,omitempty" json:"host,omitempty"`
+	Delete    bool   `yaml:"delete,omitempty" json:"delete,omitempty"`
```

Comment:
Could you explain this in:
https://github.com/lima-vm/lima/blob/fa053a3d7c439b5f363eb49df0ea123d628bd33e/examples/default.yaml#L378-L383

Reply:
`DeleteOnStop` might be more descriptive